Routine/Service/ServiceHttpHandler.cs

```diff
@@ -33,69 +32,57 @@ public class ServiceHttpHandler : IHttpHandler, IRouteHandler
 		public static string FileAction => "File";
 		public static string FontsAction => "Fonts";
 
+		#endregion
 
-		private static IJsonSerializer CreateDefaultSerializer(int maxResultLength)
-		{
-			return new JavaScriptSerializerAdapter(new JavaScriptSerializer
-			{
-				MaxJsonLength = maxResultLength,
-				RecursionLimit = DEFAULT_RECURSION_LIMIT
-			});
-		}
+		#region Constructor
 
-		public ServiceHttpHandler(IServiceContext serviceContext) : this(serviceContext, CreateDefaultSerializer(serviceContext.ServiceConfiguration.GetMaxResultLength()))
-		{
-			this.serviceContext = serviceContext;
-		}
+		private readonly IServiceContext serviceContext;
+		private readonly IJsonSerializer jsonSerializer;
 
 		public ServiceHttpHandler(IServiceContext serviceContext, IJsonSerializer jsonSerializer)
 		{
 			this.serviceContext = serviceContext;
 			this.jsonSerializer = jsonSerializer;
+
 			modelIndex = new Dictionary<string, List<ObjectModel>>();
 		}
 
-		#region Interface Implemntations
-		public bool IsReusable => false;
+		#endregion
 
-		public IHttpHandler GetHttpHandler(RequestContext requestContext)
-		{
-			return this;
-		}
+		#region Interface Implementations
+		public bool IsReusable => false;
 
-		public void ProcessRequest(HttpContext context)
+		public void ProcessRequest(HttpContext httpContext)
```

Comment:
bu dosya ve ServiceRouteHandler için genel olarak seninle bir refactoring seansı yapalım, temiz olsun